src/main/java/com/uid2/admin/audit/QLDBAuditWriter.java

```diff
@@ -37,20 +36,35 @@ public void writeLog(AuditModel model) {
         }
         try {
             if (qldbLogging) {
-                qldbDriver.execute(txn -> {
-                    List<IonValue> sanitizedInputs = new ArrayList<>();
-                    JsonObject jsonObject = model.writeToJson();
-                    StringBuilder query = new StringBuilder("UPDATE " + logTable + " AS t SET data = ?");
-                    sanitizedInputs.add(ionSys.newLoader().load(jsonObject.toString()).get(0));
-                    query.append(" WHERE t.data.itemType = ? AND t.data.itemKey = ?");
-                    sanitizedInputs.add(ionSys.newString(jsonObject.getString("itemType")));
-                    sanitizedInputs.add(ionSys.newString(jsonObject.getString("itemKey")));
+                JsonObject jsonObject = model.writeToJson();
+                if(jsonObject.getString("actionTaken").equals("CREATE")){
```

Comment:
Also why not have one method for created table and one for write logs? I thought you only need to create in your initialization function?

Reply:
We need to create a new entry in the event that for example a new admin is created; it must now have a corresponding QLDB entry